components/ai_chat/resources/page/components/main/index.tsx

```diff
@@ -63,6 +63,9 @@ function Main() {
   let currentErrorElement = null
 
   let scrollerElement: HTMLDivElement | null = null
+  let headerElement: HTMLDivElement | null = null
+  let conversationContentElement: HTMLDivElement | null = null
```

Comment:
**note:** normally in React you can't write things this imperatively - however, because we're passing a new `ref` function into the component every render, this works for these.

https://legacy.reactjs.org/docs/refs-and-the-dom.html#caveats-with-callback-refs

(sorry, just explaining why I've commented to `useRef` on the variables you added below but not these ones)